src/lightning_app/core/work.py

```diff
@@ -78,6 +79,9 @@ def __init__(
             local_build_config: The local BuildConfig isn't used until Lightning supports DockerRuntime.
             cloud_build_config: The cloud BuildConfig enables user to easily configure machine before running this work.
             run_once: Deprecated in favor of cache_calls. This will be removed soon.
+            start_before_setup: Whether the work should be started before the event loop is triggered.
```

Comment:
I'm not sure this is needed strictly speaking. Couldn't we have a `setup` method and then just call `start` from there?
I don't think we gain much from having this flag, from a startup time perspective, since flow needs to be provisioned anyway before we warmup the works.

Reply:
ok, let's just rename to `start`

Reply:
This is required to ensure we can start the works at the same time as the flow.

Reply:
Yes, this was my original name. Changed back. The only caveat is the start argument and the start method.